packages/ui/src/components/Table/Table.tsx

```diff
@@ -41,8 +41,8 @@ export const Table = memo(({ /*useTableElement = true, */ bare, ...props }: Tabl
 
 	return (
 		<UseTableElementContext.Provider value={/*useTableElement*/ true}>
-			<FieldContainer label={!bare && props.heading}>
-				<Box className="view-no-padding">
+			<FieldContainer label={!bare && props.heading} useLabelElement={false}>
```

Comment:
why is FieldContainer required in the first place?

Reply:
It adds heading above the table box when present.